### PR TITLE
Closes #147: Hotfix consensus page loading

### LINKED ISSUE #147: User reports consensus page crashing
User reports:
Everything is going fine with the two assessments we have up so far.

 

When I try to look at it my old consensus form last year’s: I get this error and crashes the browser:

A script on this page may be busy, or it may have stopped responding. You can stop the script now, open the script in the debugger, or let the script continue.

 

Script: http://pdredesign.org/assets/p…43241386530ce46ae399528c079.js:23052

Users consensus assessments have over 100 participants.

If you need email address, let me know.

Browser Details:
Operating System	Windows Windows 7
Screen Resolution	1024 x 768
Web Browser	Firefox 44.0
Browser Size	1024 x 634
IP Address	67.23.111.2
Color Depth	16
Javascript	Enabled
Flash Version	21.0.-1
Cookies	Enabled
User Agent	Mozilla/5.0 (Windows NT 6.1; WOW64; rv:44.0) Gecko/20100101 Firefox/44.0

### PR DESCRIPTION
**Pull request by MakotoTheKnight**
This PR is ready to go. Notes:

 - I did my best to make as few structural changes this time around, since a lot of that structural work has already happened on staging. This is to help smooth out the merge down the road.

### CODE REVIEW

**Comment by MakotoTheKnight** [comment]
@juliosantos:  Could I get you to confirm this PR?

File: spec/controllers/v1/responses_controller_spec.rb [hunk 1]
```diff
@@ -97,7 +97,96 @@
...
+      expect(json["id"]).to eq(42)
+    end
 
+    it "doesn't allow non-owners to view response" do
+      sign_in @user3
+      get :show_slimmed, assessment_id: assessment.id, response_id: 42
+      assert_response :forbidden
+    end
+
+    it 'returns the category => question => score struct' do
```
Comment (juliosantos) [inline spec/controllers/v1/responses_controller_spec.rb]: Probably a domain misunderstanding, but how are you testing the ` => question => score struct` of this?

**Comment by juliosantos** [comment]
LGTM. Here's a thought though.

I wonder whether this:
`authority_actions evidence: 'read'`
Would be better placed after the `evidence` method. The authority gem allows you to call `authority_actions` as many times as you like in the same controller. It would be more consistent with how I normally use `alias`, also under the method in question, closer to where it’s relevant.
I don’t have a strong opinion on this, but thought I’d bring it up to your consideration.
(in `app/controllers/v1/consensus_controller.rb`)

(same would go for `authority_actions show_slimmed: :read` in `app/controllers/v1/responses_controller.rb`)

File: spec/controllers/v1/responses_controller_spec.rb (hunk 1 above)
Comment (MakotoTheKnight) [inline spec/controllers/v1/responses_controller_spec.rb]: This is a holdover from the original test suite in that the endpoint I created should not differ in functional operation from what already exists.

**Pull request merged**